axum-macros/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-10-20
```

Comment:
Is there any difference to having this file here vs in the root of the workspace?

Reply:
I choose to follow the same strategy that I use for diesel. Only the compile tests should use a nightly compiler, anything else should use the default tool chain. By putting the rust-toolchain file into this sub-directory it's only applied for running tests from inside this directory.